packages/docs/docs/theme/default-theme-config.md

```diff
@@ -557,6 +557,90 @@ layout: SpecialLayout
 
 This will render `.vuepress/components/SpecialLayout.vue` for the given page.
 
+## Code Groups and Code Blocks
```

Comment:
Can we add a notice for the release this will come out in? 

This is a good example.

https://vuejs.org/v2/guide/components-slots.html#Scoped-Slots

Feel free to add a placeholder that I'll replace when I do the release.